### PR TITLE
Angola

### PR DESCRIPTION
**Pull request by SHAMSUDDIN-CELL**
Dear Lubos, Here is the updated file with Portuguese language and tax codes

### CODE REVIEW

File: Angola [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+  },
+  "79f99d26-e43a-4ecb-a9c9-0774601a9b2e": {
+    "79f99d26-e43a-4ecb-a9c9-0774601a9b2e": {
+      "8": 1,
+      "14": true
+    }
+  },
+  "26b9e4a5-ce10-4f30-94c7-23a1ca4428f9": {
+    "1bfdfb92-a84e-487e-b4c9-9719e9bac833": {
+      "1": "Advertising and promotion",
```
Comment (lubos) [inline Angola:24]: Account names are still in English. They need to be renamed to Portugees.

File: Angola [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+    "43271d7b-6a2f-40fc-a8b7-433955c58c88": {
+      "1": "Funds contributed"
+    },
+    "a343b383-b1c2-4acb-892d-43bd88db153e": {
+      "1": "Share of profit"
+    }
+  },
+  "7f368d97-8b7f-4b39-b156-dc66afd9496a": {
+    "42257efd-9d1a-4844-b0e6-a80f564b5332": {
+      "1": "IVA - Taxa Reduzida (2%)",
```
Comment (lubos) [inline Angola:119]: Any reason for tax code to be named `IVA - Taxa Reduzida (2%)` instead of just `IVA 2%` ?

File: Angola (hunk 1 above)
Comment (SHAMSUDDIN-CELL) [reply Angola:24]: you mean need all account information in portuguese

File: Angola (hunk 2 above)
Comment (SHAMSUDDIN-CELL) [reply Angola:119]: on the invoice you only need to show the 2% VAT is enough

File: Angola (hunk 1 above)
Comment (SHAMSUDDIN-CELL) [reply Angola:24]: I can add additional accounts too?

Comment (lubos) [reply Angola:24]: You can but only if they are really common in Angola

File: Angola (hunk 2 above)
Comment (lubos) [reply Angola:119]: Tax code should be named based on what should show on invoice. You can create sample invoice in your accounting file and submit sample data too.

**Pull request closed without merging**